ui/app/common/patient-search/constants.js

```diff
@@ -5,8 +5,8 @@ Bahmni.Common.PatientSearch = Bahmni.Common.PatientSearch || {};
 Bahmni.Common.PatientSearch.Constants = {
     searchExtensionTileViewType: "tile",
     searchExtensionTabularViewType: "tabular",
-    tabularViewIgnoreHeadingsList: ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment"],
-    identifierHeading: ["ID", "Id", "id", "identifier", "DQ_COLUMN_TITLE_ACTION"],
+    tabularViewIgnoreHeadingsList: ["display", "uuid", "image", "$$hashKey", "activeVisitUuid", "hasBeenAdmitted", "forwardUrl", "programUuid", "enrollment", "extraIdentifier"],
```

Comment:
It appears that `extraIdentifier` is a specific key used in the CURE implementation and not a default requirement for the product. As far as I understand please do correct me if I'm mistaken, the list of keys depends on the `emrapi.sqlSearch.activePatientsByLocation` query, which is configurable. For example, if we add a "gender" field to the query and it returns an object with "gender" in it, the current implementation doesn't address how to handle this. 

Presently, `tabularViewIgnoreHeadingsList` is a constant array, and adjusting it requires modifying the `openmrs-module-bahmniapps` source code. I believe it would be beneficial to configure this instead, allowing implementers to have full control over displayed content when modifying `emrapi.sqlSearch.activePatientsByLocation`.

cc: @Arjun-Go, @angshu, @binduak, @gokultw

Reply:
@rahu1ramesh create a card to move this as a config for the later point of time

Reply:
@binduak @Arjun-Go It's essential to address the hardcoding of `tabularViewIgnoreHeadingsList` and `identifierHeading`, considering `extraIdentifier` and `Patient ID` is specific to the Cure implementation. This could lead to unexpected changes in either the product or CURE. Please prioritize resolving this to ensure that implementation-specific code isn't hardcoded.

cc: @parvathy00, @angshu, @gokultw

Reply:
Change has been made as part of this commit - https://github.com/Bahmni/openmrs-module-bahmniapps/pull/628/commits/48f91045b673418e6dddaa51f33a7de6b972a10e